### PR TITLE
[MechanicalLoad] Implement buildStiffnessMatrix for TaitSurfacePressureForceField

### PR DESCRIPTION
**Pull request by alxbilger**
A task from https://github.com/sofa-framework/sofa/issues/3967

______________________________________________________

By submitting this pull request, I acknowledge that  
**I have read, understand, and agree [SOFA Developer Certificate of Origin (DCO)](https://github.com/sofa-framework/sofa/blob/master/CONTRIBUTING.md#sofa-developer-certificate-of-origin-dco)**.
______________________________________________________

**Reviewers will merge this pull-request only if**  
- it builds with SUCCESS for all platforms on the CI.
- it does not generate new warnings.
- it does not generate new unit test failures.
- it does not generate new scene test failures.
- it does not break API compatibility.
- it is more than 1 week old (or has fast-merge label).

### CODE REVIEW

**Comment by alxbilger** [comment]
[ci-build][with-all-tests]

File: Sofa/Component/MechanicalLoad/src/sofa/component/mechanicalload/TaitSurfacePressureForceField.inl [hunk 1]
```diff
@@ -396,6 +398,64 @@ void TaitSurfacePressureForceField<DataTypes>::addKToMatrixT(const core::Mechani
     }
 }
 
+template <class DataTypes>
+void TaitSurfacePressureForceField<DataTypes>::buildStiffnessMatrix(sofa::core::behavior::StiffnessMatrix* matrix)
```
Comment (epernod) [inline Sofa/Component/MechanicalLoad/src/sofa/component/mechanicalload/TaitSurfacePressureForceField.inl:402]: don't know if this is relevant but I see you copy past the content of the method addKToMatrixT and not addKToMatrix like what you did in other components.

**Review by epernod — COMMENTED** [review]
Everyday I discover a new SOFA component with your work...

File: Sofa/Component/MechanicalLoad/src/sofa/component/mechanicalload/TaitSurfacePressureForceField.inl (hunk 1 above)
Comment (alxbilger) [reply Sofa/Component/MechanicalLoad/src/sofa/component/mechanicalload/TaitSurfacePressureForceField.inl:402]: You are right. In this force field, `addKToMatrix` calls `addKToMatrixT`, but it is hidden by several layers (`BlocMatrixWriter`). See https://github.com/sofa-framework/sofa/blob/e4b94df31793bbe4b8ef62dd7412f26b6a07d18f/Sofa/framework/Core/src/sofa/core/behavior/BlocMatrixWriter.h#L157

**Review by epernod — APPROVED** [review]

**Pull request merged**